src/main/java/Task.java

```diff
@@ -0,0 +1,9 @@
+public class Task {
+    public String taskName;
+    public boolean completed = false;
+
+    Task(String taskName) {
+        this.taskName = taskName;
+        completed = false;
```

Comment:
If you are assigning completed to false here, perhaps you don't need to do it in the declaration again